mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -963,7 +963,7 @@ func resourceMongoDBAtlasClusterImportState(d *schema.ResourceData, meta interfa
 	d.SetId(encodeStateID(map[string]string{
 		"cluster_id":    u.ID,
 		"project_id":    *projectID,
-		"cluster_name":  *name,
+		"cluster_name":  u.Name,
```

Comment:
Same as other comment - need to call out if we had to fix the actual resource for the test to pass (that means it is a fixed bug)

Reply:
Same answer as another comment because is a test acceptance that imports, there's a weird behavior with terraform that sometimes fails sometimes doesn't and the resource that happen can vary.

Reply:
So when you made this change the import stopped failing?   Or you made the change and it didn't fix it?

Reply:
It didn't make a difference, I mean locally usually works because I run one test acc, that's why I assume it's related with running many testacc at same time but can't be sure, in fact, when I run make testacc locally, it happened the error for import in some resources

Reply:
If it didn't make a difference it's best to not leave it in unless the fix makes sense for another reason.

Reply:
btw we do rate limit the API, just realized ... maybe you are running into that?  You should get a response code: https://docs.atlas.mongodb.com/api/index.html#rate-limiting

Reply:
It didn't appear that error, I think it's related to ParallelTest but can't be sure but only happens when there are many testacc running at same time, right now it only has one import I think from paco(threshold attribute related) and it doesn't fail because there are not other similar testacc running at same time I think